ur_robot_driver/test/switch_on_test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+import sys
+import time
+import unittest
+
+
+import rospy
+import rostopic
+
+
+PKG = 'ur_rtde_driver'
```

Comment:
So you've had these since way back then eh ? ;)

Reply:
Damnit! I did a lot of rebasing this morning, but I have missed that one.

Reply:
It's in one of the other Python scripts as well.

Reply:
Fixed in 9a96d9f